wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -1,406 +1,355 @@
 
 Hold **Alt** to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
-| A Measure                        | Divisor | Beat              | Ticks notes                                        | Notes                                                      |
-|----------------------------------|---------|-------------------|----------------------------------------------------|------------------------------------------------------------|
-| ![](BSD 1 1b.jpg "BSD 1 1b.jpg") | 1/1     | Full Beats        | White ticks only                                   | Best used for Easy difficulty.                             |
-| ![](BSD 1 2.jpg "BSD 1 2.jpg")   | 1/2     | Half Beats        | Adds red ticks                                     | Best used for Easy/Normal difficulties.                    |
-| ![](BSD 1 3.jpg "BSD 1 3.jpg")   | 1/3     | Triplets\*        | Has two purple ticks between each white tick       | Compound timing - songs which use this are rare.           |
-| ![](BSD 1 4.jpg "BSD 1 4.jpg")   | 1/4     | Quarter Beats     | Adds blue ticks, in addition to the red ticks      | Preference for general mapping, no matter what difficulty. |
-| ![](BSD 1 6.jpg "BSD 1 6.jpg")   | 1/6     | Double Triplets\* | Has two purple ticks between each red tick         | Compound timing - same as 1/3.                             |
-| ![](BSD 1 8.jpg "BSD 1 8.jpg")   | 1/8     | Eighth Beats      | Adds yellow ticks, in addition to the red and blue | Not recommended unless you are very experienced.           |
+#### Divisor 1/1
+
+![Divisor 1/1](/wiki/shared/BSD_1_1b.jpg "Divisor 1/1")
+
+- Full Beats
+- White ticks only
+- Best used for Easy difficulty.
+
+#### Divisor 1/2
+
+![Divisor 1/2](/wiki/shared/BSD_1_2.jpg "Divisor 1/2")
+
+- Half Beats
+- Adds red ticks
+
+#### Divisor 1/3
+
+![Divisor 1/3](/wiki/shared/BSD_1_3.jpg "Divisor 1/3")
+
+- Triplets*
+- Has two purple ticks between each white tick
+- Compound timing
+
+#### Divisor 1/4
+
+![Divisor 1/4](/wiki/shared/BSD_1_4.jpg "Divisor 1/4")
+
+- Quarter Beats
+- Adds blue ticks, in addition to the red ticks
+- Useful for general mapping.
+
+#### Divisor 1/6
+
+![Divisor 1/6](/wiki/shared/BSD_1_6.jpg "Divisor 1/6")
+
+- Double Triplets*
+- Has two purple ticks between each red tick
+- Compound timing
+
+#### Divisor 1/8
+
+![Divisor 1/8](/wiki/shared/BSD_1_8.jpg "Divisor 1/8")
+
+- Eighth Beats
+- Adds yellow ticks, in addition to the red and blue
 
 **Insert Break Time** button will start/stop "Break Time" in current timestamp and **x/y** is current mouse coordinate.
 
-Centre-Left (Selector)
-----------------------
+### Centre-Left (Selector)
+
+![Hit Notes Selector](BE_HO.jpg "Hit Notes Selector")
 
-<img src="BE_HO.jpg" title="fig:Hit Notes Selector" alt="Hit Notes Selector" width="200" /> **Your construction toolkit, made for DIY.**
+**Your construction toolkit, made for DIY.**
 
 **Please note that in Standard/osu! 's default notes hitsound works different with osu!mania**
```

Comment:
change line to `**Please note that in osu!standard's default notes hit sounds works differently in osu!mania**`